includes/admin/class-give-settings.php

```diff
@@ -262,15 +262,15 @@ public function give_settings( $active_tab ) {
 						),
 						array(
 							'name'    => esc_html__( 'Currency', 'give' ),
-							'desc'    => esc_html__( 'Choose your currency. Note that some payment gateways have currency restrictions.', 'give' ),
+							'desc'    => esc_html__( 'The donation currency. Note that some payment gateways have currency restrictions.', 'give' ),
 							'id'      => 'currency',
 							'type'    => 'select',
 							'options' => give_get_currencies(),
 							'default' => 'USD',
 						),
 						array(
 							'name'    => esc_html__( 'Currency Position', 'give' ),
-							'desc'    => esc_html__( 'Choose the position of the currency sign.', 'give' ),
+							'desc'    => esc_html__( 'The position of the currency sign.', 'give' ),

```

Comment:
Correction: "The position of the currency symbol." -- Reference: https://en.wikipedia.org/wiki/Currency_symbol